beetsplug/subsonicupdate.py

```diff
@@ -51,12 +53,18 @@ def __init__(self):
         self.register_listener('import', self.loaded)
 
     def loaded(self):
+        schema = 'http://'
+        # DEPRECATED host, port, user, passw, contextpath
+        # use ${url}
         host = config['subsonic']['host'].as_str()
         port = config['subsonic']['port'].get(int)
         user = config['subsonic']['user'].as_str()
         passw = config['subsonic']['pass'].as_str()
         contextpath = config['subsonic']['contextpath'].as_str()
 
+        url = config['subsonic']['url'].as_str()
```

Comment:
I think this new setting will need a default in the `config['subsonic'].add(...)` call.